src/lib/ui/noise-sensor/NoiseThresholdsList.tsx

```diff
@@ -0,0 +1,140 @@
+import type { NoiseSensorDevice, NoiseThresholds } from 'seamapi'
+import { ZonedTime } from 'zoned-time'
+
+import { formatTime, formatTimeZone } from 'lib/dates.js'
+import { ArrowRightIcon } from 'lib/icons/ArrowRight.js'
+import { useNoiseThresholds } from 'lib/seam/noise-sensors/use-noise-thresholds.js'
+import { DetailRow } from 'lib/ui/layout/DetailRow.js'
+import { DetailSection } from 'lib/ui/layout/DetailSection.js'
+import { DetailSectionGroup } from 'lib/ui/layout/DetailSectionGroup.js'
+
+interface NoiseThresholdsListProps {
+  device: NoiseSensorDevice
+}
+
+export function NoiseThresholdsList({
+  device,
+}: NoiseThresholdsListProps): JSX.Element {
+  const { noiseThresholds, isInitialLoading } = useNoiseThresholds({
```

Comment:
Why are we passing `isInitialLoading` around instead of showing a loading state for the entire section?

Are we using or can we use the loading behavior / loading toasts like we do elsewhere? It's ok if we address this as a follow up PR, but I thought we had standardized the component loading behavior.

Reply:
I wanted to avoid an additional ternary operator on `Content`. But that's a good point, I can add the loading toast as well. There's no design for this, and it will overlay the device card at the top. I'll address this in a subsequent PR if you don't mind.